modules/$.string-pad.js

```diff
@@ -0,0 +1,50 @@
+// http://wiki.ecmascript.org/doku.php?id=strawman:string_padding
+'use strict';
+var $def = require('./$.def')
+  , $ = require('./$')
+  , repeat = require('../fn/string/repeat');
+
+module.exports = function(left) {
+  return function $pad(that, minLength, fillChar) {
```

Comment:
you mean just the `$pad`, with an extra parameter to denote left or right padding?